src/common/units.jl

```diff
@@ -21,14 +18,8 @@ E.g. `unit_to_au(:eV)` returns the conversion factor from electron volts to Hart
 """
 unit_to_au(symbol::Symbol) = getfield(units, symbol)
 
-# Convert a temperature to an energy via the Boltzmann constant.
-function uconvert(a::Unitful.EnergyUnits, x::Unitful.Quantity{T,Unitful.𝚯,U}) where {T <: Real, U <: Unitful.Units}
-    uconvert(a, Unitful.k * x)
-end
-
 # Convert a value to an energy in atomic units.
 # If given a number, assume it is already in atomic units (pass through directly).
-function to_energy(x::Unitful.Quantity)::Real
-    ustrip(uconvert(UnitfulAtomic.Eh_au, x))
+function to_energy(x)
+    austrip(x)
 end
```

Comment:
There is not really much point in having this function any more is there? But maybe I miss something.